### PR TITLE
Tech/2538/load gz files via url parameter

### PR DESCRIPTION
**Pull request by Hall-Ma**
# Load CodeCharta map via URL parameter on the demo website

Closes: #2538

See also previous PR #2917

### CODE REVIEW

**Comment by Hall-Ma** [comment]
@ce-bo Do you have an idea why we don't load the compressed files on our website? What have we missed?

**Review by ce-bo — APPROVED** [review]
Yeah, just the links were outdated. Thank you for updating them.

LGTM :-)

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ assignees: ''
 
 ## Expected Behavior
 
-<GIVEN>
+GIVEN
```
Comment (ce-bo) [inline .github/ISSUE_TEMPLATE/---bug-report.md:14]: Why has this been changed? Please revert, if it was changed unintentionally.

Comment (Hall-Ma) [reply .github/ISSUE_TEMPLATE/---bug-report.md:14]: Because Sonarlint threw an error that the tag was not closed and therefore I couldn't commit it.

**Pull request merged**